templates/schema.ldif.j2

```diff
@@ -0,0 +1,397 @@
+dn: cn=subschemasubentry,ou=schema
+objectclass: metaSchema
+objectclass: top
+cn: subschemasubentry
+m-dependencies: system
+m-dependencies: inetorgperson
+m-dependencies: core
+
+dn: ou=attributetypes,cn=subschemasubentry,ou=schema
+objectclass: organizationalUnit
+objectclass: top
+ou: attributetypes
+
+dn: m-oid=1.1.1.1.1,ou=attributetypes,cn=subschemasubentry,ou=schema
+objectclass: metaAttributeType
+objectclass: metaTop
+objectclass: top
+m-oid: 1.1.1.1.1
+m-name: tipOfTheDay
+m-equality: caseIgnoreMatch
+m-syntax: 1.3.6.1.4.1.1466.115.121.1.15
+
+dn: m-oid=1.1.1.1.2,ou=attributetypes,cn=subschemasubentry,ou=schema
+objectclass: metaAttributeType
+objectclass: metaTop
+objectclass: top
+m-oid: 1.1.1.1.2
+m-name: showThroughcareKeyDates
+m-equality: caseIgnoreMatch
+m-syntax: 1.3.6.1.4.1.1466.115.121.1.15
+
+dn: m-oid=1.1.1.1.3,ou=attributetypes,cn=subschemasubentry,ou=schema
+objectclass: metaAttributeType
+objectclass: metaTop
+objectclass: top
+m-oid: 1.1.1.1.3
+m-name: bypassDiaryPicklist
```

Comment:
Do these vars actually behave, yaml/python can sometimes be funny with `-` normally safer to use an undescore